test-cases/longevity/longevity-multidc-parallel-topology-schema-changes-12h.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 800
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(strategy=NetworkTopologyStrategy,eu-west1scylla_node_west1=3,eu-west2scylla_node_west2=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,eu-west1scylla_node_west=3,eu-west2scylla_node_west=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
```

Comment:
Add a c-s user profile with MV (simple one that builds a view on the main table keyspace1.standard1".